app/services/process_payment_intent.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
```

Comment:
I think here, as the class doc, is the perfect place to explain why we need to process a payment intent so we shed some light on this async algorithm.

Reply:
fde6c19f3922c7dad36e3cf498541b13c209320c 👍